### DISCUSSION TITLE
Plots Not Found

### DISCUSSION THREAD

**Discussion by eNaimi-dt3cc** [Chia-Network/chia-blockchain#11675] [Support]
The plots will be removed from the software list after working for a few days, and the software must be closed and reopened for the plots to be reloaded (the plots will not be loaded by clicking refresh)

**Comment by eNaimi-dt3cc** [comment]
The plots will be removed from the software list after working for a few days, and the software must be closed and reopened for the plots to be reloaded (the plots will not be loaded by clicking refresh)

only in Chia Version 1.3 - 1.3.5 - Windows 10 64bit

**Comment by loppefaaret** [comment]
how many plots do you have loaded on the client ?
